README.md

```diff
@@ -0,0 +1,69 @@
+- 📗 Table of Contents
+- 📖 About the Project
+- 🛠 Built With
+- Key Features
+- 💻 Getting Started
+- Setup
+- Prerequisites
+- Install
+- Usage
+- Run tests
+- Deployment
+- 👥 Authors
+- 🔭 Future Features
+- 🤝 Contributing
+- ⭐️ Show your support
+- 🙏 Acknowledgements
+- ❓ FAQ
+- 📝 License
+# SETUP AND MOBILE VERSION PORTFOLIO WEBSITE
+
+## 📖 About the Project
+- The main purpose of this project is to setup a portfolio for mobile version devices by use of flexbox
+## Built With
+- HTML/CSS linters
+## Key Features
+ - There are two sections:
+    - The toolbar(Header section)
+    - The headline section (right after the header)
+    
+ # 💻 GETTING STARTED
+ 
+ ## Setup:
+ Vscode set up properly
+ 
+## Install:   
+   - Git bash
+   - liveserver
+   - Install repository on Vscode
+## Usage 
+  - For Microverse Assignment
+
+## Deployment
+ - Deployed to GitHub
+ 
+ ## 👥 Author
+  - VITOESI ELIJAH
```

Comment:
- [ ] Please make sure you replace your name with the `Author` like this :point_down: 
```md
 ## 👥 VITOESI ELIJAH
```